src/tests/class_tests/openms/source/RNaseDigestion_test.cpp

```diff
@@ -184,6 +187,53 @@ START_SECTION((void digest(IdentificationData& id_data, Size min_length = 0,
 }
 END_SECTION
 
+START_SECTION((bool getVariableInosine()))
+{
+  RNaseDigestion rd;
+  TEST_EQUAL(rd.getVariableInosine(), false);
+  rd.setVariableInosine(true);
+  TEST_EQUAL(rd.getVariableInosine(), true);
+}
+END_SECTION
+
+START_SECTION((void setVariableInosine(bool value)))
+{
+  // if the test fails, make sure the right enzyme definition is loaded:
+  // const RNase* rnase = RNaseDB::getInstance()->getEnzyme("RNase_T1");
+  // STATUS(rnase->getCutsAfterRegEx());
+
+  IdentificationData id_data;
+  IdentificationData::ParentMolecule rna("test", IdentificationData::MoleculeType::RNA, "pAUGUCGCAG");
+  id_data.registerParentMolecule(rna);
+
+  RNaseDigestion rd;
+  rd.setEnzyme("RNase_T1"); // cuts after G and leaves a 3'-phosphate
+  rd.setVariableInosine(true); // optional cuts after I (modified from A)
+  rd.digest(id_data);
+
+  TEST_EQUAL(id_data.getIdentifiedOligos().size(), 7);
+
+  rd.setMissedCleavages(1);
+  rd.digest(id_data);
+
+  TEST_EQUAL(id_data.getIdentifiedOligos().size(), 11);
+
+  // get oligos ordered by occurrence in the RNA:
```

Comment:
what about this comment? should this go into a Tutorial_ ?

Reply:
I've used this for debugging and left it in there for when things break again in the future. ;-)
I've updated the comment accordingly.